glossary.html

```diff
@@ -24,46 +24,58 @@
             </nav>
         </header>
         <main>
             <h1>Glossary</h1>
-            <dl>CLASS - HTML elements can have one or more classes, separated by spaces. You can style elements using
-                CSS by selecting them with their classes.</dl>
-            <dl>ID - An HTML element can have an id attribute to identify it. id elements should always be unique to
-                that single element, and each element should never have more than one id.</dl>
-            <dl>HREF - Links tell the browser where to go using an href attribute, which stores a URL.</dl>
-            <dl>Basic Formatting - You can easily format text to be bold, italic, or underlined using simple formatting
-                tags.</dl>
-            <dl>Body - The body is the container for all of a page’s content. Comes after the
+            <dl>
+                <dt>CLASS</dt>
+                <dd>HTML elements can have one or more classes, separated by spaces. You can style elements using
+                    CSS by selecting them with their classes.</dd>
 
-                <head> tag, within the overall <html> tag.</dl>
+                <dt>ID</dt>
+                <dd>An HTML element can have an id attribute to identify it. id elements should always be unique to
+                    that single element, and each element should never have more than one id.</dd>
+
+                <dt>HREF</dt>
+                <dd>Links tell the browser where to go using an href attribute, which stores a URL.</dd>
+
+                <dt>Basic Formatting</dt>
+                <dd>You can easily format text to be bold, italic, or underlined using simple formatting
+                    tags.</dd>
+
+                <dt>Body</dt>
+                <dd>The body is the container for all of a page’s content. Comes after the
+
+                    &lt;head&gt; tag, within the overall &lt;html&gt; tag.</dd>
+            </dl>
         </main>
         <footer>
             <hr>
             <blockquote>
-                <p>"To give style to one's character - a great and rare art! He exercises it who surveys all
+                "To give style to one's character - a great and rare art! He exercises it who surveys all
                     that his nature presents in strength and weakness and then moulds it to an artistic plan until
                     everything
-                    appears as art and reason, and even the weaknesses delight the eye." </p>
-                <cite><span class="author"> -Friedrich Nietzsche </span></cite>
+                    appears as art and reason, and even the weaknesses delight the eye."<br>
+                <cite>&mdash;<span class="author">Friedrich Nietzsche </span></cite>
             </blockquote>
-            <p>© 2020 <a href="https://www.facebook.com/terry.metcalfe3"> Terrence Metcalfe </p></a>
+            <p>© 2020 <a href="mailto:t@email.com?subject=Message%20from%20CSD120%20Website"> Terrence Metcalfe </a></p>
```

Comment:
mailto URL
Make sure you close your nested elements before you close the parent elements (use a validator!)